apps/docs/libraries/ts/ratelimit/override/overview.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Overview"
+description: "Ratelimit overrides"
+---
+
+
+Ratelimit overrides are a way to override the ratelimit for specific users or group using an identifier.
+
+## Configure your override
+
+```ts
+import { Override } from "@unkey/ratelimit"
+
+const unkey = new Override({
+  rootKey: process.env.UNKEY_ROOT_KEY,
+})
+```
+
+## Use it
+
+```ts
+async function handler(request) {
+
+  const identifier = request.getUserId() // or ip or anything else you want
+
+  const override = await unkey.setOverride({
+      identifier: identifier,
+      limit: 10,
+      duration: 60000,
+      namespaceName: "email.outbound",
+      async: true
+  })
+  if (override.error){
+    // handle the error here
+    throw new Error(override.error.message);
+    return;
```

Comment:
This is invalid now, after it throws the code execution stops

either throw or return, not both